docs/setup-development.md

```diff
@@ -0,0 +1,83 @@
+# Installation
+
+Clone the repo:
+
+    git clone git@github.com:darokel/bento.git
+
+
+## Setup the application
+
+Run the following commands:
+
+```bash
+yarn
+bundle
+bundle exec rails db:setup
+```
```

Comment:
I think it'd be useful to have something about populating the local database correctly too

Reply:
db:setup runs `seed` with some test data but have included some steps to seed a sanitised dump